plugin-vidarr/src/main/java/ca/on/oicr/gsi/shesmu/vidarr/VidarrProvenanceResponseDto.java

```diff
@@ -0,0 +1,41 @@
+package ca.on.oicr.gsi.shesmu.vidarr;
+
+import com.fasterxml.jackson.annotation.JsonIgnoreProperties;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.databind.JsonNode;
+import java.util.Set;
+
+/** Bean of provenance response from Vidarr */
+@JsonIgnoreProperties(ignoreUnknown = true)
+public class VidarrProvenanceResponseDto {
```

Comment:
Seems like a duplicate of https://github.com/oicr-gsi/vidarr/blob/master/vidarr-pluginapi/src/main/java/ca/on/oicr/gsi/vidarr/api/AnalysisProvenanceResponse.java ?